searchwidget/src/main/java/com/harsh/searchwidget/Model/AnalyticsModel.kt

```diff
@@ -0,0 +1,13 @@
+package com.harsh.searchwidget.Model
+
+/**
+ * A Kotin Data class that holds data pertaining to Analytics
+ * Removes verbosity introduces due to getters and setters in Java
```

Comment:
Remove this line everywhere. I guess this is not needed.

Reply:
My first PR didn't have it. Cadacy suggested that classes need documentation, otherwise it wasn't accepting my PR.